### PR TITLE
Feature/geode 1983

### PR DESCRIPTION
**Pull request by kjduling**
Updated the spring security configuration to support the Swagger2 resources.
Precheckin successful.

### CODE REVIEW

File: geode-assembly/src/test/java/org/apache/geode/rest/internal/web/SwaggerVerificationTest.java [hunk 1]
```diff
@@ -42,6 +44,7 @@
   static Properties properties = new Properties() {
     {
       setProperty(START_DEV_REST_API, "true");
+      setProperty(SECURITY_MANAGER, SampleSecurityManager.class.getName());
```
Comment (jinmeiliao) [inline geode-assembly/src/test/java/org/apache/geode/rest/internal/web/SwaggerVerificationTest.java]: using SamplesecurityManager will require you to use a json file as well. Use SimpleSecurityManager instead.

**Comment by kjduling** [comment]
Updated to use SimpleSecurityManager.

**Comment by kirklund** [comment]
Travis failed with:
* What went wrong:
Execution failed for task ':geode-core:spotlessJavaCheck'.
> Format violations were found. Run 'gradlew spotlessApply' to fix them.
      geode-core/src/main/java/org/apache/geode/internal/cache/wan/parallel/ParallelQueueRemovalMessage.java
      geode-core/src/test/java/org/apache/geode/internal/cache/wan/parallel/ParallelQueueRemovalMessageJUnitTest.java
      geode-core/src/test/java/org/apache/geode/internal/cache/BucketRegionQueueHelper.java

Have you run precheckin yet?

**Comment by kjduling** [comment]
I didn't modify those classes.  I merged in from develop, which means this code is not formatted correctly in develop.

**Comment by kjduling** [comment]
Travis is happy now.

**Pull request closed without merging**